src/visualization.cpp

```diff
@@ -22,10 +22,8 @@ cv::Point2d worldToCanvas(const geo::Vector3& p)
 
 // ----------------------------------------------------------------------------------------------------
 
-void visualize(const World& world, Id robot_id, bool collision = false, bool show_full_map = false, Bbox centerframe = {-1e5, -1e5, 1e5, 1e5})
+void visualize(const World& world, const std::vector<Robot*>& robots, bool collision = false, bool show_full_map = false, Bbox centerframe = {-1e5, -1e5, 1e5, 1e5})
```

Comment:
Now we cannot see which of the robots collided. I think we should give each robot its own color in visualization and in terminal output?

Reply:
That is a little hard to do since we only keep a boolean of whether or not we have collided. The information of which robot it was is lost